mne/io/pick.py

```diff
@@ -192,7 +192,7 @@ def _triage_fnirs_pick(ch, fnirs):
 def _check_meg_type(meg, allow_auto=False):
     """Ensure a valid meg type."""
     if isinstance(meg, str):
-        allowed_types = ['grad', 'mag', 'planar1', 'planar2']
+        allowed_types = ['grad', 'mag', 'planar1', 'planar2', 'ref_meg']
```

Comment:
We need to check to see where `_check_meg_type` is used to see if this affects anything else. If it's only used in the ICA code it's probably fine, if it's used elsewhere we have to look for side effects.

If it is used elsewhere, you can add a `allow_ref_meg=False` default, and when it's used in ICA call `_check_meg_type(..., allow_ref_meg=ref_meg)`